__tests__/integration.test.js

```diff
@@ -135,4 +135,62 @@ describe("/api/articles/:article_id", () => {
     })
   })
 
-});
\ No newline at end of file
+});
+
+
+// api/articles/:article_id/comments
+
+describe("/api/articles/:article_id/comments", () => {
+
+  test.only("GET: Requests to this endpoint should respond with an array of all the comments with the requested article_id, each with the following properties; comment_id, author, body, votes, created_at, article_id", () => {
+    return request(app)
+    .get("/api/articles/3/comments")
+    .expect(200)
+    .then(({body})=>{
+      const {comments} = body
+      expect(comments.length).toBe(2)
+      expect(comments).toBeSortedBy("created_at", {descending: true})
+      comments.forEach(comment => {
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          article_id: 3,
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        })
+      })
+    })
+  })
+
+  test("GET EMPTY: When there are no comments for the requested article, request should still return an empty array", () => {
+    return request(app)
+    .get("/api/articles/2/comments")
+    .expect(200)
+    .then(({body})=>{
+      const {comments} = body
+      expect(comments).toEqual([])
+    })
+  })
+
+  test("GET 404: Requests that return no results due to a non-existent ID should return a 404 error with a relevant message", () => {
+    return request(app)
+    .get("/api/articles/666/comments")
+    .expect(404)
+    .then(({body})=>{
+      const {msg} = body
+      expect(msg).toBe("artist_id not found")
```

Comment:
article_id instead of artist?

Reply:
Yep, fixed that right after the push. 'Artist' had engrained itself in my brain from the lecture 😆